scripts/sdk_process/patch_models.py

```diff
@@ -232,6 +232,7 @@ def find_autorest_generated_folder(module_prefix="azure.mgmt"):
         'azure.mgmt.network',
         'azure.mgmt.monitor',
         'azure-mgmt-loganalytics'
+        'azure-mgmt-rdbms'
```

Comment:
To CLI team: Why does `azure-mgmt-rdbms` contain dashes (`-`) instead of dots (`.`)? We need a unified and verified convention.

Reply:
Thanks for pointing this out!